modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -5,21 +5,19 @@
 [id="ols-creating-the-credentials-secret-using-cli_{context}"]
 = Creating the credentials secret using the command line
 
-Create a file that is associated with the secret key used to access the API of your LLM provider.
+Create a file that is associated with the secret key used to access the API of your LLM provider. You can use API tokens to authenticate your LLM provider. Additionally, {azure-official} supports authentication using Entra ID.
 
 .Prerequisites
 
-* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a secret to store the Provider tokens.
-
 * You have access to the {ocp-short-name} CLI (oc).
```

Comment:
Slightly out of scope since it's not modified here, but there is an attribute for the `oc` CLI - I believe it's {oc-first}. It should be used to make sure `oc` is formatted correctly here.

Reply:
Created OLS-978 to act on this and do a global replace in OLS docs.